src/main/webapp/scripts/session-script.js

```diff
@@ -1,46 +1,80 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
+ * An array of who is currently in the session.
+ * @type {Array}
+ */
+let currentAttendees = [];
 
 /**
- * function openSessionInfo() displays the div container
- * that has information about the session.
+ * Represents the current session, a Session object.
+ * @type {Object}
  */
-function openSessionInfo() {
-  document.getElementById('session-info-div').style.display = 'block'; 
-}
+let session;
+
+/**
+ * Represents (in miliseconds) how long users are alerted of any membership
+ * changes in the session. 
```

Comment:
This desciption doesnt match the variable name. Display cadence sounds like its how often youre going to show the user something. Could you elaborate on what exactly this variable is for?

Reply:
I guess it's more so a period than a cadence. I think DISPLAY_PERIOD_MS would make more sense since it's only supposed to be a brief lapse of time instead of repeat.